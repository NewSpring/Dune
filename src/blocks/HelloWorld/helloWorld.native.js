// @flow
import { Text, View } from "react-native";
import React from "react";
import { helloStyle } from "./";
import { Auth } from "../Biometrics";

export const HelloWorld = () => (
  <View>
    <Text style={helloStyle.resolve()}>Hello World</Text>
    <Auth />
  </View>
);
HelloWorld.displayName = "HelloWorld";

export default HelloWorld;
