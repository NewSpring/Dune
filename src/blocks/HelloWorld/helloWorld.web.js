// @flow
import { helloStyle } from "./";

export const HelloWorld = () => (
  <h1 style={helloStyle.resolve()}>Hello World</h1>
);

export default HelloWorld;
