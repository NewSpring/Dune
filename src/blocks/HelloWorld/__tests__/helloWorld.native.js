import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import HelloWorld from "../helloWorld.native.js";

const generateComponent = additionalProps => (
  <HelloWorld {...additionalProps} />
);

describe("Hello World Native", () => {
  it("should render the Hello World component for native", () => {
    const tree = shallow(generateComponent());
    expect(tree).toBeDefined();
    expect(shallowToJson(tree)).toMatchSnapshot();
  });
});
