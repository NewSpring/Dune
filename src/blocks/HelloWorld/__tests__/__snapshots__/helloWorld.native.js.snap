// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello World Native should render the Hello World component for native 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "marginTop": 16,
      "paddingTop": 32,
    }
  }
  testId="helloWorld"
>
  Hello World
</Text>
`;
